lib/api/routes/matches/list.ts

```diff
@@ -1,11 +1,7 @@
 import { NextApiRequest as Req, NextApiResponse as Res } from 'next';
 
-import {
-  MatchJSON,
-  MatchesQuery,
-  MatchesQueryURL,
-  isMatchesQueryURL,
-} from 'lib/model';
+import { MatchJSON } from 'lib/model/match';
```

Comment:
Again, these imports should be sorted based on our ESLint rules. Your IDE should be warning you about this.